src/go/k8s/config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: vectorized/redpanda-operator
-  newName: vectorized/redpanda-operator
-  newTag: latest
```

Comment:
This change could be changed to `redpandadata/redpanda-operator`, but not `localhost`.

Reply:
What keeps causing this change.

Reply:
I do not understand why this keeps happening, sorry.